### PR TITLE
Low trust serverless clusters now have network access

### PR DESCRIPTION
**Pull request by gemma-shay**
https://cockroachlabs.atlassian.net/browse/DOC-4712

Remove all warnings about Serverless clusters without billing information on file being unable to access cloud storage.
Add release note.

### CODE REVIEW

**Comment by alyshanjahani-crl** [comment]
Changes LGTM!

I'll ping you once the actual change is merged, released and enabled in Cockroach Cloud (tentatively by end of next week)

File: cockroachcloud/run-bulk-operations.md [hunk 1]
```diff
@@ -108,7 +104,7 @@ Read the [`IMPORT`](../{{site.current_cloud_version}}/import.html) page for more
 
 ### Export data out of {{ site.data.products.db }}
 
-Using `EXPORT` with `userfile` is not recommended. If you need to export data from a {{ site.data.products.serverless }} cluster, you can either [set up billing for your organization](billing-management.html) to access cloud storage or export data to a local CSV file by using [`cockroach sql --execute`](../{{site.current_cloud_version}}/cockroach-sql.html#general). For example:
+Using `EXPORT` with `userfile` is not recommended. If you want to export data to a local CSV file, you can use [`cockroach sql --execute`](../{{site.current_cloud_version}}/cockroach-sql.html#general). For example:
```
Comment (rmloveland) [inline cockroachcloud/run-bulk-operations.md:107]: Can I access cloud storage even if I don't have billing set up for my organization?

In other words, can I use `EXPORT` to e.g. an S3 bucket now even though I did not input a credit card?

This seems to imply that the only thing I can do is export data to a local CSV file using `cockroach sql --execute`.  Is that the case?

Comment (rmloveland) [inline cockroachcloud/run-bulk-operations.md:107]: The mention of `EXPORT` could be a link to the [`EXPORT` docs](https://www.cockroachlabs.com/docs/stable/export.html) to show examples etc IMO

Comment (rmloveland) [inline cockroachcloud/run-bulk-operations.md:107]: The mention of `userfile` could be a link to https://www.cockroachlabs.com/docs/stable/use-userfile-for-bulk-operations.html

File: cockroachcloud/serverless-faqs.md [hunk 2]
```diff
@@ -155,9 +155,9 @@ There are some features of CockroachDB that are unsupported or partially support
...
+Yes, you can [run bulk operations on {{ site.data.products.serverless }} clusters](run-bulk-operations.html). You can run bulk operations using cloud storage providers or set up a [`userfile`](../{{site.current_cloud_version}}/use-userfile-for-bulk-operations.html) location for bulk operations.
 
-We don't recommend `userfile` for `EXPORT` operations. You can either add billing information to your organization to enable access to cloud storage, or export data to a local CSV file by using [`cockroach sql --execute`](../{{site.current_cloud_version}}/cockroach-sql.html#general). For example:
+We don't recommend `userfile` for `EXPORT` operations. You can either use cloud storage or export data to a local CSV file by using [`cockroach sql --execute`](../{{site.current_cloud_version}}/cockroach-sql.html#general). For example:
```
Comment (rmloveland) [inline cockroachcloud/serverless-faqs.md:160]: Same comments as above re: mentions of `EXPORT` and `userfile` could be links to those docs

File: cockroachcloud/serverless-faqs.md [hunk 3]
```diff
@@ -168,18 +168,10 @@ $ cockroach sql \
...
-
-{{site.data.alerts.callout_info}}
-Running a [bulk operation](run-bulk-operations.html) to cloud storage from a {{ site.data.products.serverless }} cluster without first entering billing information will cause the following error: `external network access is disabled`.
-{{site.data.alerts.end}}
+The [**Backups** page](backups-page.html) allows you to restore your cluster from automatic full cluster backups, which are performed hourly and stored for 30 days. {{ site.data.products.db }} does not take incremental backups of {{ site.data.products.serverless }} clusters, or allow database or table level restores from automatic full cluster backups. However, you can also backup and restore your {{ site.data.products.serverless }} cluster manually. You can [take backups locally](run-bulk-operations.html#backup-and-restore-data) to `userfile` or [backup to cloud storage](run-bulk-operations.html#backup-and-restore-data).
```
Comment (rmloveland) [inline cockroachcloud/serverless-faqs.md:177]: The mention of 'restore' could link to https://www.cockroachlabs.com/docs/cockroachcloud/backups-page.html

The mention of 'full cluster backups' could link to https://www.cockroachlabs.com/docs/v22.1/take-full-and-incremental-backups#full-backups

The mention of `userfile` could be a link to https://www.cockroachlabs.com/docs/stable/use-userfile-for-bulk-operations.html

File: cockroachcloud/run-bulk-operations.md (hunk 1 above)
Comment (gemma-shay) [reply cockroachcloud/run-bulk-operations.md]: updated to clarify you should be able to use cloud storage here too, but i'm not sure why we only had the local example here. talking to Kathryn and will open a separate ticket to add an example here.

**Comment by gemma-shay** [comment]
discovered conflicts with this PR being merged soon: https://github.com/cockroachdb/docs/pull/15808
pausing to coordinate

**Comment by gemma-shay** [comment]
will be followed by https://github.com/cockroachdb/docs/pull/16129

File: cockroachcloud/run-bulk-operations.md (hunk 1 above)
Comment (rmloveland) [reply cockroachcloud/run-bulk-operations.md]: SGTM, thanks!

**Pull request merged**